samples/bookinfo/routing-rules/route-rule-ratings-test-abort.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: ratings-route
+spec:
+  hosts:
+  - ratings
+  http:
+  - match:
+    - headers:
+        cookie:
+          regex: "^(.*?;)?(user=jason)(;.*)?$"
+    fault:
```

Comment:
We were thinking to disable faults for this version of v1alpha3 Virtual Service. The issue is upstream, due to how faults associate themselves with the upstream cluster rather than individual routes. What should we do instead? cc @rshriram

Reply:
There are several tasks in the docs that set delays (e.g., https://github.com/istio/istio.github.io/blob/599930db27aac1edc001dda886640e4df922c9d2/_docs/tasks/traffic-management/fault-injection.md).

Would the v1alpha1 rules still work? If so, I guess we could leave those tasks using v1alpha1 for now.

Reply:
Yes, but I'm not sure how the two are going to interplay. I'm not sure about the merge semantics between the two sets of rules, e.g. how does precedence work with the virtual service routes.

Reply:
I am working to get it enabled ASAP